hack/jenkins/e2e.sh

```diff
@@ -147,10 +145,6 @@ GCE_SOAK_CONTINUOUS_SKIP_TESTS=(
     "Skipped"
     )
 
-GCE_RELEASE_SKIP_TESTS=(
```

Comment:
Let's leave this array, but keep it empty - it would be easier to add something to here.